doc/classes/CameraServer.xml

```diff
@@ -45,6 +45,12 @@
 			</description>
 		</method>
 	</methods>
+	<members>
+		<member name="monitoring_feeds" type="bool" setter="set_monitoring_feeds" getter="is_monitoring_feeds" default="false">
+			Whether the server is actively monitoring camera feeds.
+			This costs resources, so only set it to [code]true[/code] when you're actively accessing the camera.
```

Comment:
```suggestion
			If [code]true[/code], the server is actively monitoring camera feeds.
			This has a performance cost, so only set it to [code]true[/code] when you're actively accessing the camera.
```
or
```suggestion
			If [code]true[/code], allows the server to monitor the camera feeds.
			This has a performance cost, so only set it to [code]true[/code] when you're actively accessing the camera.
```
or similar.
The description doesn't really explain what "monitoring" entails, though.

Reply:
> The description doesn't really explain what "monitoring" entails, though.

You're right. I changed it to `If [code]true[/code], the server is actively monitoring available camera feeds.`. I think it makes that a lot clearer :)